tasks/natural-language-processing/sentiment-analysis/conda.yaml

```diff
@@ -0,0 +1,24 @@
+name: produvia_env
+channels:
+  - pytorch
```

Comment:
Please only add packages which are needed for current sentiment analysis task and remove rest of them. Please refer [conda.yaml example](https://mlflow.org/docs/latest/tutorial.html#packaging-the-training-code)